package.json

```diff
@@ -14,14 +14,14 @@
     "update-actions-build": "node actions/build.js"
   },
   "dependencies": {
-    "@actions/core": "^1.2.4",
-    "@actions/github": "^2.2.0",
-    "axios": "^0.19.2",
-    "dotenv": "^6.0.0",
-    "google-auth-library": "^6.0.0",
+    "@actions/core": "^1.2.6",
+    "@actions/github": "^4.0.0",
```

Comment:
I think this update led to a breaking change and that's why the tests are failing